app/controllers/survey_controller.rb

```diff
@@ -1,4 +1,10 @@
 class SurveyController < ApplicationController
   def index
+    @questionList = Question.order(:order)
+    #@questions = [{:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}, {:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}]
+    @questionList.all.each do |theQuestion|
```

Comment:
for loop iterators should be pretty simple, replace 'theQuestion' with 'q' or something, it'll help us be more consistent going forward.